### PR TITLE
Bump line2d

### PR DESCRIPTION
**Pull request by dfcreative**
Fixes https://github.com/plotly/plotly.js/issues/2335

* [x] add test

### CODE REVIEW

File: test/image/mocks/gl2d_line_aligned.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            "y" : [ 30, 35 ],
+            "type" : "scatter",
+            "mode" : "lines+markers",
+            "name" : "All Different scatter"
+        },
+
+        {
+            "x" : [ 15, 20 ],
+            "y" : [ 23, 23 ],
+            "type" : "scattergl",
```
Comment (etpinard) [inline test/image/mocks/gl2d_line_aligned.json]: Baseline of the year so far :tada:

**Comment by etpinard** [comment]
Related commits: https://github.com/dfcreative/regl-line2d/commit/8aecf49e725fdf414f7855446bfdc6a0a9a87ba3 and https://github.com/dfcreative/regl-line2d/commit/4a9e721d2cfe5ce817974e99cc7643c14127ab13

**Comment by etpinard** [comment]
Good stuff @dfcreative 

Would you mind regenerating the package-lock file? Thanks!

**Comment by dfcreative** [comment]
@etpinard done :)

**Comment by etpinard** [comment]
Nicely done :dancer:

**Pull request merged**